src/installer/tests/Assets/TestUtils/TestProjects.targets

```diff
@@ -6,25 +6,25 @@
 
   <!-- TODO: Remove when the SDK understands the net8.0 tfm. -->
   <PropertyGroup>
-    <NETCoreAppMaximumVersion>8.0</NETCoreAppMaximumVersion>
+    <NETCoreAppMaximumVersion>9.0</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <KnownFrameworkReference Include="Microsoft.NETCore.App"
-                             DefaultRuntimeFrameworkVersion="7.0.0"
-                             LatestRuntimeFrameworkVersion="7.0.0"
+                             DefaultRuntimeFrameworkVersion="9.0.0"
+                             LatestRuntimeFrameworkVersion="9.0.0"
```

Comment:
These versions refer to the latest available targeting and runtime pack, i.e. 8.0.0-rc1....